### DISCUSSION TITLE
Can't disable NSFW

### DISCUSSION THREAD

**Discussion by jb11381** [tteck/Proxmox#281] [General]
First, thank you for this script and all the others!
I'm having trouble finding where to disable the NSFW feature. I've tried setting PHOTOPRISM_UPLOAD_NSFW: "true" in /photoprism/docker-compose.yml, but that didn't do it.  Any suggestions?
Thanks!

**Comment by tteck** [comment]
This isn't a Docker install, it's bare-metal. Look at `/var/lib/photoprism/.env`

**Reply by jb11381** [reply]
That makes sense. I'll take a look there. Thank you.

**Reply by bub4** [reply]
writing 
PHOTOPRISM_UPLOAD_NSFW='true'
in
/var/lib/photoprism/.env
with nano does the job.

Thank you for the hint @tteck

**Comment by tteck** [comment]
For a comprehensive list of configuration options, visit https://docs.photoprism.app/getting-started/config-options/
